Releasing.md

```diff
@@ -128,117 +164,130 @@ For the body of the post, just copy this checklist and again replace all occurre
+<p>o In gutenberg-mobile, create a branch from <code>trunk</code> with a name like <code>merge_release_X.XX.X_to_develop</code> and open PR to <code>develop</code>. If there are any merge conflicts, merge <code>develop</code> into the PR and resolve them.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Merge the Gutenberg PR to <code>trunk</code>.</p>
+<p>o Merge the gutenberg PR to <code>trunk</code>.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Update the Gutenberg reference in the Gutenberg Mobile PR so it points to the Gutenberg PR merge commit in <code>trunk</code>.</p>
+<p>o Update the <code>gutenberg</code> reference in the gutenberg-mobile <code>merge_release_X.XX.X_to_develop</code> PR so it points to merge commit in gutenberg <code>trunk</code> for the gutenberg PR merged in the previous task.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Merge the Gutenberg Mobile PR to <code>develop</code>.</p>
+<p>o Merge the gutenberg-mobile PR to <code>develop</code>.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:heading {"level":3} -->
-<h3>AFTER the main apps have cut their release branches</h3>
+<h3>Clean Up Pending Work</h3>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph -->
-<p>o Update the <code>gutenberg/after_X.XX.X</code> branches and open a PR against <code>develop</code>. If the branches are empty we’ll just delete them. The PR can actually get created as soon as something gets merged to the after_X.XX.X branches.&nbsp; Merge the <code>gutenberg/after_X.XX.X</code> PR(s) only AFTER the main apps have cut their release branches.</p>
+<p>⚠️ This section may only be completed <em>after</em> the main apps cut their own release branches. </p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Update the <a href="https://docs.google.com/spreadsheets/d/15U4v6zUBmPGagksHX_6ZfVA672-1qB2MO8M7HYBOOgQ/edit?usp=sharing">Release Incident Spreadsheet</a> with any fixes that occurred after the release branches were cut.</p>
+<p>o Update the <code>gutenberg/after_X.XX.X</code> branches and open a PR against <code>develop</code>. If the branches are empty we’ll just delete them. The PR can actually get created as soon as something gets merged to the <code>gutenberg/after_X.XX.X</code> branches. Merge the <code>gutenberg/after_X.XX.X</code> PR(s) only <em>AFTER</em> the main apps have cut their release branches.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:heading {"level":3} -->
-<h3>You're done</h3>
+<h3>Finish the Release</h3>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph -->
-<p>o Pass the baton. Ping the dev who is responsible for the next release</p>
+<p>o Update the <a href="https://docs.google.com/spreadsheets/d/15U4v6zUBmPGagksHX_6ZfVA672-1qB2MO8M7HYBOOgQ/edit?usp=sharing">Release Incident Spreadsheet</a> with any fixes that occurred after the release branches were cut.</p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>o Celebrate!</p>
+<p>o Pass the baton. Ping the dev who is responsible for the next release.</p>
```

Comment:
I'm wondering if we should move this earlier (i.e. to before the next release wranglers should start smoke testing). Do you think it would be worth having the previous release wranglers "pass the baton" by doing something like posting a schedule for the upcoming release in Slack? I'm just trying to think what we can do to better make sure that the "new" release wranglers are aware of what they need to do, and I was thinking that the "previous" release wranglers probably know that better than anyone. Just a thought.

Reply:
@mchowning absolutely. I believe we are actively running into that confusion now. The current rotation responsibilities actually last 3 weeks, with first week overlapping with the previous wrangler and the last week overlapping with the next wrangler.